ui/src/viz/vizContainer.tsx

```diff
@@ -71,6 +80,11 @@ export function VizContainer(props: VizContainerProps) {
             config,
             data,
             plugin: getVizPlugin(config.plugin, config.pluginConfig),
+            tablePlugin: getVizPlugin("core/table", {
```

Comment:
This table visualization will apply to all future visualizations also, not just the existing bar chart -- is that right?

Reply:
Yeah